sites/aviationpros.com/server/graphql/fragments/content-page.js

```diff
@@ -7,6 +7,10 @@ fragment ContentPageFragment on Content {
   teaser(input: { useFallback: false, maxLength: null })
   body
   published
+  siteContext {
+    path
+    canonicalUrl
```

Comment:
As far as I can tell, the `canonicalUrl` value is not being used - this can be removed.

Reply:
pushed